docs/08-developer-guide/04-project/02-release-guide.md

```diff
@@ -426,6 +479,10 @@ git tag -s "ozone-$VERSION" -m "Ozone $VERSION release"
 git push origin "ozone-$VERSION"
 ```
 
+### Update GitHub Releases page
+
+add the release information [GitHub Releases](https://github.com/apache/ozone/releases/new) page. See the [1.4.1 Release page](https://github.com/apache/ozone/releases/tag/ozone-1.4.1) for an example.
```

Comment:
`add the release information GitHub Releases page` does not seem to be a correct sentence.